### DISCUSSION TITLE
Autoplayer on profiles and feed?

### DISCUSSION THREAD

**Discussion by thalliumoxide** [cvzi/Bandcamp-script-deluxe-edition#334] [Ideas]
When searching through other peoples bandcamps, have it where the script would autoplay peoples collection, from there you can go to purchase, download, add to wishlist etc.

Also autoplayer on feed allowing you to do the same as above?

**Comment by cvzi** [comment]
Not sure what exactly you mean, maybe a screenshot would help me understand

**Comment by thalliumoxide** [comment]
![image](https://github.com/cvzi/Bandcamp-script-deluxe-edition/assets/134213470/98360fd7-d8e1-4882-96e5-ed3ba3beea90)
For example, when you click on someone's collection, instead of the standard Bandcamp player your player would be at the bottom instead. This will autoplay the tracks on someone's collection and allow you to add to wishlist etc. without you having to click on individual songs. Similarly, with your own feed have your autoplayer at the bottom for easy access at adding to wishlist etc.

**Comment by cvzi** [comment]
Don't know when I'll have time to add it though. I have created an issue to track this and remind me: https://github.com/cvzi/Bandcamp-script-deluxe-edition/issues/355

On the feed it is already possible in a way: there should be a "play album" link below each post that opens the player.

**Comment by cvzi** [comment]
This should work now.
